Gemfile.lock

```diff
@@ -256,6 +256,7 @@ PLATFORMS
   arm64-darwin-20
   arm64-darwin-21
   x86_64-darwin-21
+  x86_64-linux
```

Comment:
I believe I had this problem in another repo (I don't remember which one it was) and fixed it by just setting the platform to `ruby`. But I think this works too. I still don't understand what it means

Reply:
Yeah I'm not sure either... I know it changes when running `bundle install` (it has happened in my machine in some repos before) and it has to do with the architecture of the machine that runs the command. I thought it was the platforms that are supposed to run the corresponding code. I didn't know about `ruby` but after looking, that might also solve the issue... For now I will merge this and if it happens again I will try with that. Thanks!